source/content/test-global-cdn-caching.md

```diff
@@ -9,15 +9,13 @@ tags: [cache, cdn]
 
 1. Open a terminal.
 1. Enter the following command with your full Pantheon domain URL.
+    - The `-L` flag instructs curl to resend the request to the new location in the case of a 301, 302, or 303 redirect.
```

Comment:
To the point of the second checkbox in the Work/Prereqs in the description, maybe could add a second sentence here saying "Remove this flag if you are specifically concerned with cache headers on redirects"?